Classes/Driver/Version1/IndexerDriver.php

```diff
@@ -114,7 +114,7 @@ public function fulltext(NodeInterface $node, array $fulltextIndexOfNode, $targe
             [
                 // first, update the __fulltextParts, then re-generate the __fulltext from all __fulltextParts
                 'script' => '
-                    if (!ctx._source.containsKey("__fulltextParts")) {
+                    if (!(ctx._source.containsKey("__fulltextParts") && ctx._source.__fulltextParts instanceof LinkedHashMap)) {
```

Comment:
can we use instanceof "Map" here? Would be a little more stable and in line with Java APIs.